jar-infer/src/main/java/com/uber/nullaway/jarinfer/DefinitelyDerefedParamsDriver.java

```diff
@@ -161,6 +175,58 @@ private static String extractJAR(String jarPath) {
     return jarDir;
   }
 
+  /*
+   * Repack JAR archive and return path
+   *
+   */
+  private static String packJAR(String jarDir) {
+    Preconditions.checkArgument(
+        Files.isDirectory(Paths.get(jarDir)), "invalid jar directory!" + jarDir);
+    String jarPath = jarDir + ".ji.jar";
+    System.out.println("repacking " + jarPath + "...");
```

Comment:
I'd vote for not having prints anywhere on this code (we don't seem to have them in e.g. `NullAway.java`. But you can leave this here for now and just remember to strip them out when merging `jar-infer` fully into master.